spec/datetimeformat.html

```diff
@@ -687,6 +737,9 @@ <h1>Intl.DateTimeFormat.prototype.resolvedOptions ()</h1>
             1. Else, let _v_ be *undefined*.
           1. Else,
             1. Let _v_ be the value of _dtf_'s internal slot whose name is the Internal Slot value of the current row.
+          1. If the Internal Slot value of the current row is an Internal Slot value in <emu-xref href="#table-datetimeformat-components"></emu-xref>, then
```

Comment:
This seems to contradict the constructor of the DateTimeFormat which has logic to throw a type error if any of these fields are not undefined. From what i've seen chrome 83 decided not to throw a type error during construction and instead follow this logic while firefox 79 decided to throw an exception during construction.

Reply:
These internal slots are filled from the resolved pattern and not from the input arguments (cf. the "Set dateTimeFormat's internal slot whose name is the Internal Slot column of the row to p." step in InitializeDateTimeFormat). So this step is actually needed to be consistent with the "throw TypeError" steps in InitializeDateTimeFormat.

Reply:
Are we good here then? `CreateDataPropertyOrThrow` throws `TypeError`.

Reply:
From my end this looks good, @anba cleared up my misunderstanding